tests/test_clientsdaily.py

```diff
@@ -0,0 +1,69 @@
+import datetime as DT
+import pytest
+import json
+import os
+from mozetl.schemas import MAIN_SUMMARY_SCHEMA
+
+EXPECTED_INTEGER_VALUES = {
+    'active_addons_count_mean': 3613,
+    'crashes_detected_content_sum': 9,
+    'first_paint_mean': 12802105,
+    'pings_aggregated_by_this_row': 1122,
+    'search_count_sum': 1043
+}
+
+
+@pytest.fixture
+def make_frame(spark):
+    root = os.path.dirname(__file__)
+    path = os.path.join(root, 'resources',
+                        'main_summary-late-may-1123-rows-anonymized.json')
+    with open(path) as f:
+        jrows = json.load(f)
+        jrdd = spark.sparkContext.parallelize(map(json.dumps, jrows))
+        frame = spark.read.json(jrdd, MAIN_SUMMARY_SCHEMA)
+    return frame
+
+
+@pytest.fixture
+def make_frame_with_extracts(spark):
+    from mozetl.clientsdaily import rollup as R
```

Comment:
module-level import reveals a race condition in Pyspark initialization that ends up with "AttributeError: 'NoneType' object has no attribute '_jvm'"  when trying to invoke pyspark.sql.functions.sum(). If you have a clean solution for the race condition awesome otherwise I'd rather have this modest funkiness in the test module. What do you think?